app/routes/crate/version.js

```diff
@@ -15,6 +15,7 @@ export default Route.extend({
     async model(params) {
         const requestedVersion = params.version_num === 'all' ? '' : params.version_num;
         const crate = this.modelFor('crate');
+        const original_docs = crate.get('documentation');
```

Comment:
Could you please use camelcase as the rest of variables do in the javascript project (we should have a linter roule for this, I know)